public/index.html

```diff
@@ -7,36 +7,234 @@
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    
+    <section class="card-wrapper  margin-h-24 pad-both">
+        <img class="" src="../assets/4.png" alt="">
+        <h2>Multi-Post
+            Stories</h2>
+        <div class="canopy-list-wrapper">
+            <ul class="canopy-list">
+                <li><b>CANOPY</b></li>
+                <li class="card-empty-list"></li>
+                <li>Back End Dev</li>
+                <li class="card-empty-list"></li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        <div class="tech-used">
+            <span>
+                HTML
+            </span>
+            <span>
+                CSS
+            </span>
+            <span>
+                JavaScript
+            </span>
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            See projects
+        </button>
+    </section>
+
+    <section class="last-section">
+        <h1>About Myself</h1>
+        <p>
+            Hello I am a software developer! I can help you build a product , feature or website. Look through some of 
+            my work and experience! If you like what you see and have a project you need coded, do not hesitate to contact me.
+        </p>
+        <h3>LET'S CONNECT</h3>
+        <div class="social-icons pad-aside ">
+            <img src="../assets/social icons/Vector.svg" alt="">
+            <img src="../assets/social icons/Linkedin icon.svg" alt="">
+            <img src="../assets/social icons/Vector (1).svg" alt="">
+            <img src="../assets/social icons/Vector (2).svg" alt="">
+            <img src="../assets/social icons/Vector (3).svg" alt="">
+        </div>
+        <button type="button" class="see-projects-btn btn">
+            Get my resume
+        </button>
+        <div class="language">
+            <h3>Languages</h3>
+            <i class="fa fa-chevron-down"></i>
+        </div>
+        <div class="language-list">
+            <div class="html">
+                <img src="../assets/html.png" alt="">
+                <span>HTML</span>
             </div>
+
+            <div class="js">
+                <img src="../assets/js.png" alt="">
+                JavaScript
+            </div>
+            
+            <div class="css">
+                <img src="../assets/css.png" alt="">
+                <span>CSS</span>
+            </div>
+        </div>
```

Comment:
- [ ] Kindly, conosider using `ul` to wrap the list of language (e.g., "JavasScript", "HTML" and "CSS") and make sure each language wrap in a `li`. Remember you should always use `ul` and `li` to list element. This will improve greatly the semantic of your webpage 😄 which will make your website much more accessible (for users, screen readers, and search engines).